src/admin/menu/menu.const.ts

```diff
@@ -22,10 +20,29 @@ export const MENU_OVERVIEW_TABLE_COLS: TableColumn[] = [
 	{ id: 'actions', label: '' },
 ];
 
-export const MENU_ICON_OPTIONS: OptionsType<ReactSelectOption<IconName>> = Object.freeze([
```

Comment:
Ik stel voor om deze `MENU_ICON_OPTIONS` te renamen naar `ICON_OPTIONS` en in shared types te gooien, rekeninghoudend dat deze overal zal gebruikt worden as IconPicker options.